src/object/player.cpp

```diff
@@ -2090,19 +2083,16 @@ Player::draw(DrawingContext& context)
   */
 
   /* Draw Tux */
-  if (m_safe_timer.started() && size_t(g_game_time * 40) % 2)
+  if (m_safe_timer.started() && size_t(g_game_time * 40) % 2 || (Editor::is_active() || !m_visible))
```

Comment:
Would be better to move the two new conditions at the beginning (so they are checked for first) and wrap the two other ones in brackets as well.

Reply:
i can do that